Week3/homework/1-step3.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
-function foo(func) {
-  // What to do here? 
-}
+(function foo(func) {
+  console.log("Hello, Foo");
```

Comment:
Here you are supposed to call the function given as parameter, namely func. When foo bar is called ( foo(bar); in the initial file), the func parameter given to the funct function will become bar, such that when you call the func parameter, you actually call the bar function, which will to the console itself.